open_spiel/python/mfg/algorithms/mfg_proximal_policy_optimization_test.py

```diff
@@ -0,0 +1,98 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Mean field proximal policy optimaztion."""
```

Comment:
Thanks for this, I was going to ask for a basic test.

Now you need to add it to the CMakeLists though, specifcally here: https://github.com/google-deepmind/open_spiel/blob/931e39a99ee73412500def0227925f8f19f033fe/open_spiel/python/CMakeLists.txt#L291

Reply:
Added in the CMakeList under the 291 line (OPEN_SPIEL_ENABLE_PYTORCH). Actually, I changed also the name of the test file to comply with the rest of pytorch tests -> mfg_proximal_policy_optimization_pytorch_test.py